prow/plugins/lgtm/lgtm.go

```diff
@@ -201,11 +202,13 @@ func handleGenericComment(gc githubClient, config *plugins.Configuration, owners
 	} else if LGTMCancelRe.MatchString(rc.body) {
 		wantLGTM = false
 	} else {
-		return nil
+		return status, nil
 	}
 
 	// use common handler to do the rest
-	return handle(wantLGTM, config, ownersClient, rc, gc, log, cp)
+	err := handle(wantLGTM, config, ownersClient, rc, gc, log, cp)
+	status.TookAction = true
```

Comment:
Ideally we wouldn't need the author to remember to do this correctly in every single case. Since we're creating a new plugin agent and sub-classing the clients etc inside of it, could we not detect that the GitHub client we gave to a specific plugin handler for a given event was used for something?

Reply:
Does that mean return status from `handle` or else?